packages/maps/src/lib/map/Map.svelte

```diff
@@ -1 +1,87 @@
-<p>I'm a map from the map package</p>
+<script context="module">
+	export const appendOSKeyToUrl = (osKey) => {
+		return (url) => {
+			url = new URL(url);
+
+			if (!url.hostname === 'api.os.uk') {
+				return { url: url.href };
+			}
+
+			const WGS84 = 3857;
```

Comment:
This line is misleading.

`EPSG 3857` is the projected coordinate system commonly referred to as Web Mercator, whereas I think most people would expect `WGS84` to refer to `EPSG 4326`.

These are fundamentally different things: 4326 uses angular units (degrees) to represent positions on an ellipsoid; 3857 uses linear units (m) to represent positions projected onto a plane.

Reply:
I see, I've updated the naming.